lib/traject/extract_fulltext_link.rb

```diff
@@ -1,8 +1,15 @@
 module ExtractFulltextLink
+  # Filters out 856u fields that look like: http://alexandria.ucsb.edu/lib/ark:/48907/f3gt5k61
+  # We're attempting to capture URLs that look like:
+  #   http://gateway.proquest.com/openurl?url_ver=Z39.88-2004&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:dissertation&amp;res_dat=xri:pqm&amp;rft_dat=xri:pqdiss:3602190
```

Comment:
This comment doesn't seem to match the regex that is used below.  Should the regex also be checking for proquest domain?